components/Currency/CurrencyIcon/CurrencyIcon.tsx

```diff
@@ -72,6 +72,11 @@ import iTRXIcon from '@synthetixio/assets/synths/iTRX.svg';
 import iDASHIcon from '@synthetixio/assets/synths/iDASH.svg';
 import iETCIcon from '@synthetixio/assets/synths/iETC.svg';
 import sTSLAIcon from '@synthetixio/assets/synths/sTSLA.svg';
+import sFBIcon from '@synthetixio/assets/synths/sFB.svg';
```

Comment:
Surely we don't need to do this every time. There's gotta be a better way without explicitly listing the synths here. cc @andytcf